utils/firestore.js

```diff
@@ -2180,40 +2234,148 @@ const getParticipantSelection = async (filter) => {
     }
 }
 
-const assignKitToParticipants = async (data) => {
+const assignKitToParticipant = async (data) => {
     try {
-        const snapshot = await db.collection("kitAssembly").where('supplyKitId', '==', data.supply_kitId).where('supplyKitIdUtilized', '==', false).get();
-        if (Object.keys(snapshot.docs).length !== 0) {
-            snapshot.docs.map(doc => {
-                data['collection_cardId'] = doc.data().collectionCardId
-                data['collection_cupId'] = doc.data().collectionCupId
-                data['specimen_kitId'] = doc.data().specimenKitId
-                data['specimen_kit_usps_trackingNum'] = doc.data().uspsTrackingNumber
-            })
+        const snapshot = await db.collection("kitAssembly").where('690210658', '==', data['690210658']).where('221592017', '==', '517216441').get();
+        if(snapshot.size > 0) {
+            data['687158491'] = snapshot.docs[0].data()[687158491];
             const docId = snapshot.docs[0].id;
             await db.collection("kitAssembly").doc(docId).update(
-            { 
-                supplyKitIdUtilized: true
+            {
+                '531858099': data['531858099'],
+                '221592017': '241974920',
+                '418571751': '266600170',
+                'Connect_ID': parseInt(data['Connect_ID'])
             })
-            await db.collection("participantSelection").doc(data.id).update(
+            const secondSnapShot = await db.collection("participants").where('Connect_ID', '==', parseInt(data['Connect_ID'])).get();
```

Comment:
`participantSnapshot` or something more descriptive would be a nice naming convention here.